scripts/poo-tracker.coffee

```diff
@@ -0,0 +1,27 @@
+Redis = require 'redis'
+Url = require 'url'
+
+POO_TRACKER_KEY = "poo:skalnik:message"
+POO_LATEST_KEY = "poo:skalnik:latest_message"
+
+info = Url.parse process.env.POO_REDIS_URL
+redis_client = Redis.createClient(info.port, info.hostname)
+redis_clent.auth info.auth.split(":")[1] if info.auth
+
+module.exports = (robot) ->
+  setInterval ->
+    checkRedisForShit()
+  , 1000
+
+  checkRedisForShit ->
+    redis_clent.get POO_TRACKER_KEY -> (err, reply)
+      return console.error("Failed get of #{POO_TRACKER_KEY}: #{err}") if err
+      return if reply == null
+
+      robot.messageRoom "#arrakis", "We got a poo over here: #{reply}"
```

Comment:
FYI the message right now is "Hi! This is Mike! I'm currently pooping!" It's the custom message he configured. IDK if it makes sense with "We got a poo over here". Eventually I'll stick a URL on the end of whatever the custom message is.